src/main/resources/default/taglib/t/additionalActions.html.pasta

```diff
@@ -14,8 +14,8 @@
 <i:local name="actions" value="@renderToString('body')"/>
 <i:if test="isFilled(actions)">
     <div class="dropdown @class">
-        <a class="h-100 btn btn-link" style="z-index: 3; position: relative" data-toggle="dropdown" aria-haspopup="true"
-           aria-expanded="false">
+        <a class="h-100 btn btn-link text-nowrap" style="z-index: 3; position: relative" data-toggle="dropdown"
+           aria-haspopup="true" aria-expanded="false">
```

Comment:
Bei mehrzeiligen Tags finde ich i. A. folgende Formatierung mit einem Attribut pro Zeile besser, damit man nicht mal aus Versehen ein in zweiter Reihe geparktes Attribut übersieht.

```suggestion
        <a class="h-100 btn btn-link text-nowrap"
           style="z-index: 3; position: relative"
           data-toggle="dropdown"
           aria-haspopup="true"
           aria-expanded="false">
```